mmv1/templates/terraform/examples/biglake_table.tf.erb

```diff
@@ -0,0 +1,61 @@
+resource "google_biglake_catalog" "catalog" {
+    name = "<%= ctx[:vars]['catalog'] %>"
+    location = "US"
+}
+
+resource "google_storage_bucket" "bucket" {
+  name                        = "<%= ctx[:vars]['bucket'] %>"
+  location                    = "US"
+  force_destroy               = true
+  uniform_bucket_level_access = true
+}
+
+resource "google_storage_bucket_object" "metadata_folder" {
+  name    = "metadata/"
+  content = " "
+  bucket  = google_storage_bucket.bucket.name
+}
+
+
+resource "google_storage_bucket_object" "data_folder" {
+  name    = "data/"
+  content = " "
+  bucket  = google_storage_bucket.bucket.name
+}
+
+resource "google_biglake_database" "database" {
+    name = "<%= ctx[:vars]['database'] %>"
+    catalog = google_biglake_catalog.catalog.id
+    type = "HIVE"
+    hive_options {
+        location_uri = "gs://${google_storage_bucket.bucket.name}/${google_storage_bucket_object.metadata_folder.name}"
+        parameters = {
+          "owner" = "John Doe"
+        }
+    }
+}
+
+resource "google_biglake_table" "<%= ctx[:primary_resource_id] %>" {
+    name = "<%= ctx[:vars]['name'] %>"
+    database = google_biglake_database.database.id
+    type = "HIVE"
+    hive_options {
+      table_type = "MANAGED_TABLE"
+      storage_descriptor {
+        location_uri = "gs://${google_storage_bucket.bucket.name}/${google_storage_bucket_object.data_folder.name}"
+        input_format  = "org.apache.hadoop.mapred.SequenceFileInputFormat"
+        output_format = "org.apache.hadoop.hive.ql.io.HiveSequenceFileOutputFormat"
+      }
+      # Some Example Parameters.
+      parameters = {
+        "spark.sql.create.version" = "3.1.3"
+        "spark.sql.sources.schema.numParts" = "1"
+        "transient_lastDdlTime" = "1680894197"
+        "spark.sql.partitionProvider" = "catalog"
+        "owner" = "John Doe"
```

Comment:
Not a blocker but it could be nice to follow the guidance at https://developers.google.com/style/examples#example-person-names to choose a name.

Reply:
done.